src/main/java/seedu/ccacommander/logic/commands/CreateEventCommand.java

```diff
@@ -22,11 +23,13 @@ public class CreateEventCommand extends Command {
             + "Parameters: "
             + PREFIX_NAME + "NAME "
             + PREFIX_LOCATION + "LOCATION "
-            + PREFIX_DATE + "DATE "
+            + PREFIX_DATE + "DATE \n"
             + "Example: " + COMMAND_WORD + " "
```

Comment:
I think we can add the tag parameter here
```suggestion
            + PREFIX_DATE + "DATE "
            + "[" + PREFIX_TAG + "TAG]...\n"
            + "Example: " + COMMAND_WORD + " "
```

Reply:
Thanks!